src/linter-provider.ts

```diff
@@ -1,13 +1,13 @@
 import { oneLine } from "common-tags";
-import readPkgUp = require("read-pkg-up");
-import requireRelative = require("require-relative");
 import {
   LinterProviderLoadError,
   ModuleNotLinterProviderError,
   NoLinterProvidersError,
 } from "./errors";
 import { LinterFactory } from "./linter-adapter";
 import { logger } from "./logger";
+import readPkgUp = require("read-pkg-up");
+import requireRelative = require("require-relative");
```

Comment:
These where above the relative imports to group package imports before relative imports.